unit-tests/func/hw-errors/l500/l500-error-common.h

```diff
@@ -0,0 +1,85 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2021 Intel Corporation. All Rights Reserved.
+
+#include "l500/l500-private.h"
+using namespace rs2;
+
+const uint8_t l500_trigger_error_opcode = 0x5E;
+
+std::map< uint8_t, std::pair< std::string, rs2_log_severity > > build_log_errors_map()
+{
+    using namespace librealsense::ivcam2;
+
+    std::map< uint8_t, std::pair< std::string, rs2_log_severity > > res;
+    l500_notifications_types last_index = success;
+    for (auto & i : l500_fw_error_report)
+    {
+        if (i.first == success)
+            continue;  // LRS ignore if he get success as error
+
+        if (i.first > last_index + 1)
```

Comment:
You're assuming that `l500_fw_error_report` is ordered...
I think you can add a `REQUIRE( i.first > last_index );` so it would fail if not

Reply:
l500_fw_error_report is map but I will add this check